lib/glific/third_party/google_translate/translate.ex

```diff
@@ -26,19 +43,26 @@ defmodule Glific.GoogleTranslate.Translate do
     middleware
     |> Tesla.client()
     |> Tesla.post(@endpoint, data, opts: [adapter: [recv_timeout: 120_000]])
-    |> handle_response()
+    |> handle_response(non_translatable_segments)
   end
 
-  @spec handle_response(tuple()) :: tuple()
-  defp handle_response(response) do
+  @spec handle_response(tuple(), list()) :: tuple()
+  defp handle_response(response, non_translatable_segments) do
     response
     |> case do
       {:ok, %Tesla.Env{status: 200, body: %{"data" => %{"translations" => translations}}}} ->
         translated_texts =
           translations
-          |> Enum.map(fn translation -> translation["translatedText"] end)
+          |> Enum.map_join("\n", fn translation -> translation["translatedText"] end)
+
+        # Combine translated and non-translatable segments
+        combined_texts =
+          non_translatable_segments
```

Comment:
Doing this will not preserve the original order of lines. Attaching the default texts on send message and the translated hindi node. We can see after the translation the lines are not in correct order, the line in this case is `@results.result_1`
<img width="516" alt="Screenshot 2024-08-13 at 2 35 07 PM" src="https://github.com/user-attachments/assets/47fa174c-812e-4fee-86e7-6a2b90fcf413">
<img width="396" alt="Screenshot 2024-08-13 at 2 35 15 PM" src="https://github.com/user-attachments/assets/eee99304-1b98-4264-af7b-eb9d1b44bced">

Also can we add a test later which handles this scenario?